plugin.yaml

```diff
@@ -178,6 +178,10 @@ node_types:
         description: |
           Force delete vm for external resource
         default: false
+      ignore_state_changes:
```

Comment:
This doesn't seem to really describe what this flag does.

It might be better termed "power_on_existing_vms" and default to true, with a description something like "Power on already-existing VMs."

Except even then, it's not quite doing this, as this will stop someone from invoking start operations on a VM they have invoked a stop operation on, while not preventing the 'stop' in the first place (even if the VM was created by the plugin itself).

So... naming aside, I think this also has potential to cause problems in future. What problem is supposed to be solved by this, and how is anything else expected to work if a pre-existing VM is powered off and we don't power it on when we're told to use it?

Reply:
This change was proposed to solve the problem with shadow VM.
Client has blueprint that fetches the shadow VM, but those cannot be powered on (because such VMs are locked) and therefore during start operation error is thrown. 

What if we will change the name to something like "change_power_state", set it to true by default and add condition to stop() operation which won't execute "stop" operation if the parameter is set to false?

Reply:
Isn't the correct solution to this to detect when there are shadow VMs and then do something (who knows what, given that it's by definition a broken VM, but the fact that the plugin is looking at it means it's one that the deployment in question wants it to look at)?

Reply:
OK, so following discussion on slack I suggest this be set to:

enable_start_vm, defaulting to true, and with a description something like "Enable start (power on) operation for VM."

Possibly wants some rewording but I think something like that makes it more clear what it's for.